app/views/requests/_calculate_product_totals.html.erb

```diff
@@ -0,0 +1,30 @@
+<div id="calculateTotals" class="modal fade">
+  <div class="modal-dialog modal-dialog-scrollable">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h4 class="modal-title">Product Totals</h4>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body p-4">
+        <table class="table table-hover striped">
+          <thead>
+          <tr>
+            <th>Item</th>
+            <th>Quantity</th>
+          </tr>
+          </thead>
+          <tbody>
+            <% @calculate_product_totals.sort_by { |name, quantity| -quantity}.each do |name, quantity| %>
```

Comment:
This works as it is to handle sorting. I would prefer to use something like:
```
@calculate_product_totals.sort_by { |name, quantity| quantity}.reverse
```

Although this is fairly a subjective choice of styling :), so I'am fine with this approach as well.